doc/langref.html.in

```diff
@@ -5452,6 +5452,22 @@ fn doAThing(str: []u8) void {
           a default value of 13. The type of the right hand side of the binary {#syntax#}catch{#endsyntax#} operator must
               match the unwrapped error union type, or be of type {#syntax#}noreturn{#endsyntax#}.
       </p>
+     <p>
+      If you want to provide a default value with
+      {#syntax#}catch{#endsyntax#} after performing some logic, you
+      can combine {#syntax#}catch{#endsyntax#} with named {#link|Blocks#}:
+      </p>
+      {#code_begin|syntax|handle_error_with_catch_block.zig#}
+const parseU64 = @import("error_union_parsing_u64.zig").parseU64;
+
+fn doAThing(str: []u8) void {
+     const number = parseU64(str, 10) catch blk: {
+         // do things
+         break :blk 13;
+     };
```

Comment:
```suggestion
    const number = parseU64(str, 10) catch blk: {
        // do things
        break :blk 13;
    };
```
Is it possible to run zig fmt on these code blocks?

Reply:
Dunno about that! But good catch. Thanks!